### PR TITLE
Added `:all` Symbol to Javis

### LINKED ISSUE #247: Make an `:all` Symbol for Frame Ranges
**Is your feature request related to a problem? Please explain.**

Right now, one can define an Object such that,

```julia
BackgroundObject(1:200, ground) # creates the background of the animation

Object(1:100, myfunc)
Action(:same, myaction)
Action(myaction) # the same as the preceding line

Object(100:200, myotherfunc)
Action(myotheraction) # as the unit range of 100:200
```

It would be nice to have a unit range that allows an action to be defined over the entirety of the animation.

**Describe the solution you'd like**

Here is an idea:

```julia
BackgroundObject(1:200, ground) # creates the background of the animation

Object(1:100, myfunc)
Action(:same, myaction)
Action(myaction) # the same as the preceding line

Object(100:200, myotherfunc)
Action(myotheraction) # as the unit range of 100:200
Action(:all, mybestfunc) # defined over the range of 1:200 frames
```

Furthermore, the `:all` keyword would allow an Action to be defined anywhere in the code and not immediately after a prior `Action`.

### PR DESCRIPTION
**Pull request by TheCedarPrince**
## PR Checklist

If you are contributing to `Javis.jl`, please make sure you are able to check off each item on this list:

- [x] Did I update `CHANGELOG.md` with whatever changes/features I added with this PR?
- [x] Did I make sure to only change the part of the file where I introduced a new change/feature?
- [x] Did I cover all corner cases to be close to 100% test coverage (if applicable)?
- [x] Did I properly add Javis dependencies to the `Project.toml` + set an upper bound of the dependency (if applicable)?
- [x] Did I properly add test dependencies to the `test` directory (if applicable)?
- [x] Did I check relevant tutorials that may be affected by changes in this PR?
- [x] Did I clearly articulate why this PR was made the way it was and how it was made?

**Link to relevant issue(s)**
Closes #247 

**How did you address these issues with this PR? What methods did you use?**

I added to the Frames.jl and Object.jl file code to allow for finding the maximum of a background frame. I did not add tests as I saw that we have not added tests for the `:same` keyword. Should we add testing? Does the logic make sense here for what I did?

### CODE REVIEW

File: src/structs/Frames.jl [hunk 1]
```diff
@@ -51,8 +54,10 @@ function get_frames(parent, elem, frames::Symbol, last_frames::UnitRange; is_fir
             return 1:length(last_frames)
         end
         return last_frames
+    elseif frames === :all
+        return 1:maximum(CURRENT_VIDEO[1].background_frames)
     else
-        throw(ArgumentError("Currently the only symbol supported for defining frames is `:same`"))
+        throw(ArgumentError("Currently the only symbols supported for defining frames are `:same` and `:all`."))
```
Comment (Wikunia) [inline src/structs/Frames.jl:60]: we might want to use `join(FRAMES_SYMBOL, ", ", " and ")` or something like that

**Review by Wikunia — CHANGES_REQUESTED** [review]
One minor thing you might want to add and yes please add test cases. I think we actually have some for `:same`. Maybe not unit tests but we use it in some animations.

**Comment by Wikunia** [comment]
You might wanna have a look @TheCedarPrince 
Do you think this is ready for merging?

**Review by Wikunia — APPROVED** [review]

**Comment by TheCedarPrince** [comment]
Hey @Wikunia - I tested this out yesterday and all the tests passed on my end! I ran into the ImageIO issue (#315) similar to @flyaflya while testing so I will continue experimenting there. Otherwise, I approve this PR! Feels good to be getting in the groove again! :wink:

**Pull request merged**